docs/src/main/paradox/user/lang/kotlin/crdt.md

```diff
@@ -0,0 +1 @@
+# CRDT
```

Comment:
Here you could use placeholders like in the other languages not yet describing them (see JavaScript and Go)

```
Explain how to use the CRDT API
Explain default values
Explain onStateSet
Explain how to handle streamed calls
Explain the APIs for each different CRDT
```

Reply:
Ok perfect